typings/trim-canvas.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'trim-canvas' {
```

Comment:
I'll try to prepare `@types/trim-canvas` package in the next few days so this local module declaration can be replaced with DefinitelyTyped one.

Reply:
I was initially going to migrate `trim-canvas` to TS first (it's also a package of mine and much smaller surface area), and then do `react-signature-canvas`, but demand was a lot higher here so I reversed that.

It's on my TODO list anyway so I'd prefer not having an `@types/` package for it, particularly as there's no real demand for it (`trim-canvas` basically never gets issues) and this is very tiny 😅